static/routes.config.php

```diff
@@ -298,6 +299,14 @@
 		'/userexport[/{action}]' => [Module\Settings\UserExport::class,             [R::GET, R::POST]],
 	],
 
+	'/network' => [
+		'[/]'                         => [Module\Conversation\Network::class, [R::GET]],
+		'/archive/{from:\d\d\d\d-\d\d-\d\d}[/{to:\d\d\d\d-\d\d-\d\d}]' => [Module\Conversation\Network::class, [R::GET]],
+		'/forum/{contact_id:\d+}'     => [Module\Conversation\Network::class, [R::GET]],
+		'/group/{group_id:\d+}'       => [Module\Conversation\Network::class, [R::GET]],
+		'/accounttype/{accounttype}'  => [Module\Conversation\Network::class, [R::GET]],
```

Comment:
Have you changed the `accounttype` value to always have this `/accounttype/` in the path? I would suggest using the widget with the `$_GET` parameter instead.

Reply:
You can see at the start of the `content()` method that the widget have different base URLs matching these routes. And I can use the other account type widget, I figured there must have been a reason for using the path instead.

Reply:
At first I started using it with a path since there had been code on the community page for this. I already thought about replacing it with the parameter in the future, so that we only do have got a single widget.

Reply:
I agree with that, I wasn't happy to see both widgets in the wild.

Reply:
Do you want to change that for the network page in this PR?

Reply:
Yes, I'm currently at work, but I'll be able to make the change in a couple of hours.

Reply:
Done.